requirements.txt

```diff
@@ -20,12 +20,12 @@ deepspeed>=0.17.0
 trl==0.25.0
 hf_xet==1.2.0
 kernels>=0.9.0
-trackio
+trackio>=0.11.0
 
 optimum==1.16.2
 hf_transfer
 sentencepiece
-gradio==5.49.1
+gradio>=6.0,<7.0
```

Comment:
While I was at it, I upgraded the version of `gradio` to 6.0, as this is compatible with the latest version of `trackio`, and brings other benefits, e.g. Gradio 6 .whl is much smaller than the Gradio 5 .whl. I went through the code and updated the Gradio app to use Gradio 6 syntax.